Assets/Resources/OVRPlatformToolSettings.asset

```diff
@@ -0,0 +1,22 @@
+%YAML 1.1
```

Comment:
I think this one doesn't belong in git; it's now in gitignore and appeared with the move to the newest Oculus unity package. Ditto for the .meta associated

Reply:
Ah! Good catch!